extensions/amp-bind/0.1/amp-state.js

```diff
@@ -38,39 +38,35 @@ export class AmpState extends AMP.BaseElement {
   }
 
   /** @override */
+  activate(invocation) {
+    const event = invocation.event;
+    if (event && event.detail) {
+      this.updateState_(event.detail.response);
+    }
+  }
+
+  /** @override */
   buildCallback() {
     const TAG = this.getName_();
 
     toggle(this.element, false);
     this.element.setAttribute('aria-hidden', 'true');
 
-    const id = user().assert(this.element.id,
-        '%s element must have an id.', TAG);
-
-    let json;
     const children = this.element.children;
     if (children.length == 1) {
       const child = children[0];
       if (isJsonScriptTag(child)) {
-        json = tryParseJson(children[0].textContent, e => {
+        const json = tryParseJson(children[0].textContent, e => {
           user().error(TAG, 'Failed to parse state. Is it valid JSON?', e);
         });
+        this.updateState_(json, true);
```

Comment:
Add a comment to indicate what's the second `true` parameter is.  Like: `this.updateState_(json, /*opt_isInit*/ true);`